### PR TITLE
[WIP] Issue 74

### PR DESCRIPTION
**Pull request by benasher44**

### CODE REVIEW

**Comment by gregheo** [comment]
@benasher44 do you want to squash and merge this now? Or keep it open and you'll keep adding commits to it through tomorrow?

File: _drafts/2017-06-15-issue-74.md [hunk 1]
```diff
@@ -4,15 +4,18 @@ title: ! 'Issue #74'
...
+Also, keep in mind that Swift 3.2 is the Swift 4 compiler running in Swift 3 compatibility mode (`-swift-version 3`, which clicked for me sometime during WWDC 😅). Understanding this can be really helpful when putting together a ticket for [bugs.swift.org](https://bugs.swift.org).
 
 <!--excerpt-->
 
 {% include sponsor.html %}
 
 ### Starter tasks
 
-> TODO
+- [SR-4866](https://bugs.swift.org/browse/SR-4866): Stack overflow: Parsing phony empty paren exprs
```
Comment (jessesquires) [inline _drafts/2017-06-15-issue-74.md]: let's spell-out "paren exprs" 😄

**Comment by jessesquires** [comment]
looks awesome so far! 🎉

File: _drafts/2017-06-15-issue-74.md (hunk 1 above)
Comment (benasher44) [inline _drafts/2017-06-15-issue-74.md]: `paren` is a word though. So just `exprs`?

**Comment by benasher44** [comment]
@gregheo ready to squash and merge after CI! 🎉

**Pull request merged**